.github/workflows/codeql.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: codeQL
+
+on:
+  push:
+    branches: [development/*, stabilization/*, hotfix/*]
+  pull_request:
+    branches: [development/*, stabilization/*, hotfix/*]
```

Comment:
There is no "stabilization" and "hotfix" on this repository

Reply:
Will it never happen? I don't think it's bothersome to leave that in place if `stabilization` or `hotfix` appears someday in this repository. Tell me what you think @tcarmet.

Reply:
To me, it will never happen, but yes we can keep it if you want :)

Reply:
If it will never happen, it's better to remove them, otherwise it might get people confused.